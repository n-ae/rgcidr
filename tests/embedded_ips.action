192.168.1.0/24 tests/parsing_edge_cases.given
