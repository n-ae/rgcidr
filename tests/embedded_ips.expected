192.168.1.1 normal IP
url=http://192.168.1.2/path
[192.168.1.3] brackets
(192.168.1.4) parentheses  
192.168.1.6:8080 with port
192.168.1.7,other,data
server-192.168.1.8-prod
